canonical-websites/www.ubuntu.com
# 1577 search

## Done

* updated the search page to vanilla brochure theme
* changed the design slightly to reflect new theme

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- use your Canonical VPN
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/search?q=arm) note there are no 'next' or 'first' links
- test the [last page of search](http://0.0.0.0:8001/search?q=arm&limit=10&offset=300) see that it has messages and no 'next' link
- test the [second page](http://0.0.0.0:8001/search?q=arm&limit=10&offset=10) see that it has no 'Previous' link
- test a [middle page](http://0.0.0.0:8001/search?q=arm&limit=10&offset=20) see that it has all the links

## Issue / Card

Fixes #1577 
Fixes #1945
